styles/integrations.css

```diff
@@ -0,0 +1,8 @@
+.icon-integration {
```

Comment:
nitpick: I wonder if there is a better way to do it without custom css

Reply:
> nitpick: I wonder if there is a better way to do it without custom css

I don't know, but I really like to write my own styles :)